roadmap/initiatives/test_infrastructure.md

```
@@ -0,0 +1,247 @@
+**Pytest Markers:**
+`python
+@pytest.mark.unit          # Fast, isolated tests
+@pytest.mark.integration   # Tests with dependencies
+@pytest.mark.e2e          # End-to-end tests
+@pytest.mark.api          # Tests requiring API calls
+@pytest.mark.serial       # Tests that must run serially
+`
+**AAA Pattern:**
+`python
+def test_something():
+    # Arrange
```

markdownlint-cli2 (0.18.1)

Headings must start at the beginning of the line

(MD023, heading-start-left)